ddt.py

```diff
@@ -148,7 +148,10 @@ def _raise_ve(*args):
                 elif isinstance(data, list):
                     value = elem
                     test_name = mk_test_name(name, value, i)
-                add_test(test_name, func, value)
+                if isinstance(value, dict):
+                    add_test(test_name, func, **value)
+                else:
+                    add_test(test_name, func, value)
```

Comment:
Looks simple and useful, although it may break backwards compatibility.

Can you illustrate with an example/test case in `test_example.py`?